lib/fprotect/Kconfig

```
@@ -50,3 +50,10 @@ config FPROTECT_BLOCK_SIZE
 	default $(dt_node_int_prop_hex,$(DT_CHOSEN_ZEPHYR_FLASH),erase-block-size)
 	help
 	  Block size of fprotect
+
+config FPROTECT_RESERVE_HUK_ACL_SLOT
+	bool "Reserve the last ACL slot for HUK"
```

This is in a file separated from `subsys/partition_manager/Kconfig` and also outside security/crypto domain so I think we should spell it out:
```suggestion
config FPROTECT_RESERVE_HW_UNIQUE_KEY_ACL_SLOT
	bool "Reserve the last ACL slot for Hardware Unique Key"
```
or
```suggestion
config FPROTECT_RESERVE_HW_UNIQUE_KEY_ACL_SLOT
	bool "Reserve the last ACL slot for Hardware Unique Key (HUK)"
```